internal/statestore/redis_indices.go

```diff
@@ -72,6 +78,20 @@ func extractIndexFilters(p *pb.Pool) []indexFilter {
 		})
 	}
 
+	for _, f := range p.BoolEqualsFilters {
```

Comment:
Can you add some comments to explain the mappings between range filters and bool values?

Reply:
I've added a comment at the top of the file explaining the general strategy.  Does this explain everything that wasn't obvious?